go/test/endtoend/sharding/resharding/byte/resharding_byte_test.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v3
+
+import (
+	"testing"
+
+	sharding "vitess.io/vitess/go/test/endtoend/sharding/resharding"
+)
+
+// TestReShardingByte - using a VARBINARY column for resharding.
+func TestReShardingByte(t *testing.T) {
```

Comment:
I would rather have us call this something like `TestReshardingString` or even `TestReshardingVarbinary`. Byte was the v2 way of sharding using string columns, we should stop referring to that (even if we are still doing a v2 test) because string / varbinary is the use case, byte is an implementation detail.